newrelic/resource_newrelic_workload.go

```diff
@@ -58,7 +58,12 @@ func resourceNewRelicWorkload() *schema.Resource {
 						"query": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "The query.",
+							Description: "A valid entity search query, empty, and null values are considered invalid.",
```

Comment:
```suggestion
  * `query` - (Required) A valid entity search query; empty, and null values are considered invalid.
```

just to make it a little more grammatically correct